### DISCUSSION TITLE
\allowdisplaybreaks[4]跨页排公式好像用不了

### DISCUSSION THREAD

**Discussion by wocoanima** [tuna/thuthesis#954] [Q&A] [answered]
\allowdisplaybreaks[4]跨页排公式好像用不了？

**Comment by zepinglee** [comment]
给个示例？

**Reply by wocoanima** [reply]
大概就是，
```latex
\begin{align}
a\\
a\\
a\\
a\\
a\\
a\\
a\\
a\\
a\\
a\\
a\\
a\\
a\\
a
\end{align}
```
搞一个很多行的公式，如果\allowdisplaybreaks[4]可以用的话，如果一页排不下整个公式，他会自动把这个很多行的公式从某一行截断，然后排到两页上面去，但是咱们这个模板应该不行的。咱们这个模板不会截断，它会给这个公式新开一页直到这新开的一页可以一次性排下整个公式

**Reply by wocoanima** [reply]
这个github有点诡异，我没一行都是a两个斜线换行，但是好像只能显示出来a\

**Reply by Harry-Chen** [reply]
请正确使用 code block 放置代码：

\`\`\`latex
you code here
\`\`\`

**Reply by zepinglee** [reply, marked as answer]
建议给出一个可复现的完整示例。下面的示例可以正确分页。

```latex
\documentclass[degree=master]{thuthesis}
\usepackage{lipsum}
\input{thusetup}
\allowdisplaybreaks[4]
\begin{document}
\lipsum[1-3]\par
\begin{align}
  a\\
  a\\
  a\\
  a\\
  a\\
  a\\
  a\\
  a\\
  a\\
  a\\
  a\\
  a\\
  a\\
  a
\end{align}
\end{document}
```

<img width="821" alt="Screenshot 2024-05-19 at 15 49 27" src="https://github.com/tuna/thuthesis/assets/12290822/8f61c8c4-38fe-47f9-abae-632fd679162f">

**Comment by wocoanima** [comment]
不好意思，是我的问题
